include/avif/avif.h

```diff
@@ -625,6 +629,30 @@ typedef struct avifGainMapMetadataDouble
 // to the closest uint32_t fractions.
 // Returns AVIF_FALSE if some field values are < 0 or > UINT32_MAX.
 AVIF_API avifBool avifGainMapMetadataDoubleToFractions(avifGainMapMetadata * dst, const avifGainMapMetadataDouble * src);
+
+// Performs tone mapping on a base image using the provided gain map.
+// The HDR capacity (also known as HDR headroom or ratio) is the ratio of HDR to
+// SDR white brightness of the display to tone map for, in linear space.
+// 'toneMappedImage' should have the 'format' and 'depth' fields set to the desired values.
+// If non NULL, 'clli' will be filled with the light level information of the tone mapped image.
+struct avifRGBImage;
+AVIF_API avifResult avifApplyGainMap(const struct avifImage * baseImage,
```

Comment:
Many libavif public functions follow an object-oriented naming convention, in which the function name begins with the type of the first parameter. Please see if these two functions should follow this naming convention.